eclipse-scout-core/src/form/fields/groupbox/GroupBoxLayout.js

```diff
@@ -27,7 +27,7 @@ export default class GroupBoxLayout extends AbstractLayout {
   }
 
   _initDefaults() {
-    this._statusWidth = HtmlEnvironment.get().fieldStatusWidth;
+    this.fieldStatusWidth = HtmlEnvironment.get().fieldStatusWidth;
```

Comment:
This is an internal property, please use `_`-prefix.

Reply:
Changed to _fieldStatusWidth